lib/membrane/video_compositor/multiple_inputs/video_compositor/implementation.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.VideoCompositor.MultipleInputs.VideoCompositor.Implementation do
+  @moduledoc """
+  A module describing multiple input video compositor implementation type and implementing
+  functions related with implementation format.
+  """
+
+  @typedoc "Define video compositor implementation types"
+  @type implementation_t :: :ffmpeg | :opengl_cpp | :opengl_rust | :nx | module()
```

Comment:
it seems `module()` is not supported

Reply:
It is visible by the IDE and recognised as a built-in alias for `atom()`

Reply:
I misunderstood, removed `module()` from the `@type`